sh.popcorntime.PopcornTime.appdata.xml

```diff
@@ -0,0 +1,49 @@
+<!-- Copyright 2018 Ströger Florian -->
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>sh.popcorntime.PopcornTime.desktop</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
+  <name>PopcornTime</name>
+  <summary>Watch Movies and TV Shows instantly</summary>
+  ​<translation type="gettext">PopcornTime</translation>
+  <update_contact>preisschild_AT_protonmail.com</update_contact>
+  <description>
+    <p>
+        Popcorn Time is a multi-platform, free software BitTorrent client that includes an integrated media player.
+        The applications provide a free alternative to subscription-based video streaming services such as Netflix. 
+        Popcorn Time uses sequential downloading to stream video listed by several torrent websites, and third party trackers can also be added manually.
+    </p>
+  </description>
+  <content_rating type="oars-1.1">
+    <content_attribute id="violence-cartoon">moderate</content_attribute>
+    <content_attribute id="violence-fantasy">moderate</content_attribute>
+    <content_attribute id="violence-realistic">moderate</content_attribute>
+    <content_attribute id="violence-bloodshed">moderate</content_attribute>
+    <content_attribute id="violence-sexual">moderate</content_attribute>
+    <content_attribute id="violence-desecration">moderate</content_attribute>
+    <content_attribute id="violence-slavery">moderate</content_attribute>
+    <content_attribute id="violence-worship">moderate</content_attribute>
+    <content_attribute id="drugs-alcohol">moderate</content_attribute>
+    <content_attribute id="drugs-narcotics">moderate</content_attribute>
+    <content_attribute id="drugs-tobacco">moderate</content_attribute>
+    <content_attribute id="sex-nudity">moderate</content_attribute>
+    <content_attribute id="sex-themes">moderate</content_attribute>
+    <content_attribute id="sex-homosexuality">moderate</content_attribute>
+    <content_attribute id="sex-prostitution">moderate</content_attribute>
+    <content_attribute id="sex-adultery">moderate</content_attribute>
+    <content_attribute id="sex-appearance">moderate</content_attribute>
+    <content_attribute id="language-profanity">moderate</content_attribute>
+    <content_attribute id="language-humor">moderate</content_attribute>
+    <content_attribute id="language-discrimination">moderate</content_attribute>
+  </content_rating>
+  <url type="homepage">https://popcorntime.sh/</url>
+  <screenshots>
+    <screenshot type="default">https://i.imgur.com/UtKMSso.jpg</screenshot>
+    <screenshot>https://i.imgur.com/7MRrqjV.jpg</screenshot>
+    <screenshot>https://i.imgur.com/1NrtHBj.png</screenshot>    
+  </screenshots>
+  <releases>
+    <release version="0.3.10" date="2018-05-14"/>
```

Comment:
`0.3.10` was released in Oct 31, 2016 ...not a few months ago.

Reply:
The version from their CI (https://ci.popcorntime.sh/job/Popcorn-Time-Desktop/) has a few fixes and was released a few months ago. Since they have not changed the version, I went with 0.3.10, but I'm open for input to change the version number.